src/libANGLE/renderer/metal/SurfaceMtl.mm

```diff
@@ -206,6 +206,13 @@ void StopFrameCapture()
     {
         mColorFormat = display->getPixelFormat(angle::FormatID::B8G8R8A8_UNORM_SRGB);
     }
+    else if (attribs.get(EGL_GL_COLORSPACE, EGL_GL_COLORSPACE_LINEAR) == EGL_GL_COLORSPACE_BT2020_PQ_EXT)
+    {
+        mColorFormat.intendedFormatId = mColorFormat.actualFormatId =
+            angle::FormatID::R16G16B16A16_FLOAT;
+        mColorFormat.metalFormat = MTLPixelFormatRGBA16Float;
+        mColorFormat.metalColorspace = CGColorSpaceCreateWithName(kCGColorSpaceITUR_2020_PQ);
```

Comment:
I was using `mColorFormat` to hold the CGColorRef between the `SurfaceMtl()` and `SurfaceMtl::initialize()` methods. Should I add another variable in the class instead?

Also here I'm forcing RGBA16 just when the `EGL_GL_COLORSPACE` is set, which doesn't seem exactly right but I wasn't sure what is better.